net-news/liferea/liferea-1.14.0.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit autotools gnome2-utils optfeature python-single-r1 xdg
+
+DESCRIPTION="News Aggregator for RDF/RSS/CDF/Atom/Echo feeds"
+HOMEPAGE="https://lzone.de/liferea/"
+SRC_URI="https://github.com/lwindolf/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-db/sqlite:3
+	dev-libs/fribidi
+	dev-libs/glib:2
+	dev-libs/gobject-introspection
+	dev-libs/json-glib
+	dev-libs/libpeas[gtk,python,${PYTHON_SINGLE_USEDEP}]
```

Comment:
I wonder, 
```
configure: error: Package requirements (	gtk+-3.0 >= 3.22.0
		glib-2.0 >= 2.50.0
		gio-2.0 >= 2.50.0
		pango >= 1.4.0
		libxml-2.0 >= 2.6.27
		libxslt >= 1.1.19
		sqlite3 >= 3.7.0
		gmodule-2.0 >= 2.0.0
		gthread-2.0
		libsoup-2.4 >= 2.42
		webkit2gtk-4.0
		json-glib-1.0
		gobject-introspection-1.0
		gsettings-desktop-schemas
		libpeas-1.0 >= 1.0.0
		libpeas-gtk-1.0 >= 1.0.0
		fribidi >= 0.19.7) were not met:

Package 'libpeas-gtk-1.0', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables PACKAGE_CFLAGS
and PACKAGE_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```
Could you investigate where does the .pc file come from?

Reply:
Hmm that's strange how you got this error.  
The libpeas-gtk-1.0.pc comes from `dev-libs/libpeas[gtk]`. The `gtk` USE flag is sufficient to get .pc to be installed (I stripped all USE flags except `gtk` to check it).

Reply:
That IS interesting.
```
my-test-container-snap-29144 ~ # emerge -pv libpeas

These are the packages that would be merged, in order:

Calculating dependencies  ... done!

[binary   R    ] dev-libs/libpeas-1.34.0-1::gentoo  USE="gtk python -glade -gtk-doc -lua -test -vala" LUA_SINGLE_TARGET="lua5-1 -luajit" PYTHON_SINGLE_TARGET="python3_10 -python3_9 -python3_11" 0 KiB

Total: 1 package (1 reinstall, 1 binary), Size of downloads: 0 KiB

my-test-container-snap-29144 ~ # equery f libpeas | grep pkgconfig
/usr/lib64/pkgconfig
/usr/lib64/pkgconfig/libpeas-1.0.pc
```

After rebuilding the package with 
`USE="gtk python" emerge -1av libpeas --usepkg=n`
I got the file finally. So there's probably some automagic detection going on, which isn't your fault, but showed here annoyingly. 

Thanks!